packages/dev-frontend/src/components/Stability/Yield.tsx

```diff
@@ -41,6 +41,7 @@ export const Yield: React.FC = () => {
   const remainingLqtyOneYear = remainingStabilityPoolLQTYReward.mul(yearlyHalvingSchedule);
   const remainingLqtyInUSD = remainingLqtyOneYear.mul(lqtyPrice);
```

Comment:
```suggestion
  const remainingLqtyOneYearInUSD = remainingLqtyOneYear.mul(lqtyPrice);
```

Reply:
Hi @edmulraney , I've updated the names as you propose. Glad to help (even a little bit ;-))